src/tools/generate/generate_chapters.js

```diff
@@ -12,7 +12,21 @@ converter.setFlavor('github');
 converter.setOption('simpleLineBreaks', false);
 converter.setOption('tablesHeaderId', false);
 
+const min_publish_date = new Date('2019-11-05T12:00');
+const sitemap_path = `templates/sitemap.xml`;
+
+var config = {};
+var chapters_config = [];
+var todo_chapters = [];
+
 const generate_chapters = async () => {
+
+  //Read in the Chapters to see which ones are still to do (needed for sitemap)
+  await read_config('2019');
```

Comment:
I know we only have one year's config at the moment, but should this be hard coded?

Reply:
I know it's not an excuse but there is precence here 😊
https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/config.py

Reply:
Yeah it's not perfect but we can iterate on it after launch for new editions.